README.md

```diff
@@ -28,6 +28,8 @@ __Note:__ Mac OS X, Linux, and Windows are currently the only supported OSes.
       * ```sudo systemctl stop bluetooth``` (once)
       * ```sudo systemctl disable bluetooth``` (persist on reboot)
 
+If you're using BlueZ 5.14 or later, you may find that running noble *and* bleno at the same time stops you from retrieving the list of services over BLE.
+
```

Comment:
That was for a single process.

Can you please try again with the latest release of noble and bleno? There was a fix in https://github.com/sandeepmistry/node-bluetooth-hci-socket/commit/14dc3660f9e86695c6c75b92dc2222da2905549e which was released with `bluetooth-hci-socket` v0.5.1, both noble and bleno are using this release in the latest package.json's.